modules/fts/pages/fts-advanced-settings-ftsMemoryQuota.adoc

```diff
@@ -1,28 +1,18 @@
 = ftsMemoryQuota
 
-The `ftsMemoryQuota` setting controls the maximum usable memory for the FTS Search service from the total amount of RAM available in the system. 
+The `ftsMemoryQuota` setting controls the maximum usable memory, in MiB, available to the Search Service on each node in your cluster.
+This RAM allocation applies to each node that has the Search Service in 1 of its Service Groups.
 
-Sparing enough RAM memory for the filesystem cache is crucial to override the ftsMemoryQuota.
+You can configure the `ftsMemoryQuota` from the Couchbase Server Web Console at any time.
 
-The FTS Search service recommends a minimum of 512 MB memory-resident ratio for its index. However, users can spare more memory quota for a healthy resident ratio of the index. This lets the system have sufficient memory available to perform indexing, querying, or other lifecycle operations like rebalances, etc.
+If you're using the Search Service as the only Service on a node, it's recommended to set the `ftsMemoryQuota` to 80% or less of the total available RAM for that node.
+This makes sure to leave enough RAM for your operating system to manage its on-demand filesystem cache.
+The bleve text indexing library uses memory mapping for index files, which uses the operating system cache.
```

Comment:
I would drop the `operation system cache` bit - which doesn't make much sense. So re-word to this perhaps ..

```
The search process memory maps the index files to support fast access to indexed content.
```